atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresVersionCheckTest.java

```diff
@@ -32,6 +32,8 @@
     @SuppressWarnings(value = "Slf4jConstantLogMessage")
     public void shouldLogErrorOn_9_2_24() {
         Logger log = Mockito.mock(Logger.class);
+        thrown.expect(AssertionError.class);
+        thrown.expectMessage("The minimum supported version is ");
         PostgresVersionCheck.checkDatabaseVersion("9.2.24", log);
         Mockito.verify(log).error(contains("The minimum supported version is {}"), Mockito.anyObject(),
                 Mockito.eq(PostgresVersionCheck.MIN_POSTGRES_VERSION));
```

Comment:
Think we need to change this line. Using `assertAndLog` formats the string first, _and then_ later on calls `log.error("Assertion {} with exception ", message, t)` (see `AssertUtils:102`). So this call to verify will throw... but the test passes because the Mockito `ArgumentsAreDifferent` is an `AssertionError` and its string representation does contain the exception's message.